cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/RackRolling.java

```diff
@@ -740,8 +741,7 @@ public static RackRolling rollingRestart(PodOperator podOperator,
                                              int maxRestartBatchSize,
                                              KubernetesRestartEventPublisher eventPublisher) {
         //TODO: Add EventPublisher to emit kube events when restarting nodes
-
-        PlatformClient platformClient = new PlatformClientImpl(podOperator, reconciliation.namespace(), reconciliation);
+        PlatformClient platformClient = new PlatformClientImpl(podOperator, reconciliation.namespace(), reconciliation, eventPublisher);
```

Comment:
Since we use the eventPublisher when constructing PlatformClient, should we remove eventPublisher passed into RackRolling constructor (line 757)?

Reply:
Done